### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by JPLachance**
* Move the prompt style to it's own file
- Remove the btn right margin, replace it by a margin between two buttons

### CODE REVIEW

File: scss/forms/table-form.scss [hunk 1]
```diff
@@ -123,7 +123,7 @@ form {
           .dropdown-toggle {
             box-sizing: border-box;
             width: 100%;
-            padding-right: 11px;
+            padding-right: 6px;
```
Comment (marilynechan) [inline scss/forms/table-form.scss]: AFAIK, I changed this so the arrow would be aligned with the other inputs icon.

Comment (JPLachance) [inline scss/forms/table-form.scss]: I removed the `margin-right: 5px;` on the btn class which was causing your alignment issue. Yes it's aligned correctly ;)

Comment (marilynechan) [inline scss/forms/table-form.scss]: Great, just making sure ;-) .

**Comment by pochretien** [comment]
:shipit:

**Pull request merged**